apps/main/src/dex/components/PagePool/PoolDetails/CurrencyReserves/CurrencyReservesContent.tsx

```diff
@@ -64,10 +64,10 @@ const CurrencyReservesContent = ({
 
     <Box className={'right'} flex flexDirection="column">
       <Chip size="md" isBold>
-        {formatNumber(cr?.balance, { defaultValue: '-' })}{' '}
+        {!cr || isNaN(cr.balance) ? '-' : formatNumber(cr.balance, { defaultValue: '-' })}{' '}
```

Comment:
Should we not just also show the `defaultValue` as configured when the number is `NaN`?

Currently the code only works with `null`: `if (val == null) return options?.defaultValue ?? '-'`

Reply:
<img width="176" height="54" alt="image" src="https://github.com/user-attachments/assets/423bdf64-fb8a-4540-8e8a-8bfd4cb268ff" />